public/electron.js

```diff
@@ -0,0 +1,75 @@
+const path = require("path");
+
+const { app, BrowserWindow } = require("electron");
+const isDev = require("electron-is-dev");
+
+
+// Conditionally include the dev tools installer to load React Dev Tools
+let installExtension, REACT_DEVELOPER_TOOLS;
+
+if (isDev) {
+  const devTools = require("electron-devtools-installer");
+  installExtension = devTools.default;
+  REACT_DEVELOPER_TOOLS = devTools.REACT_DEVELOPER_TOOLS;
+}
```

Comment:
This code snippet is strange considering the usage later
https://github.com/ethersphere/bee-status/blob/806c48174876aa32e392cfbc467576ea532ee93e/public/electron.js#L50-L54
it seems this above snippet is the only part where you use `installExtension` and `REACT_DEVELOPER_TOOLS`.
so defining these two variables in the outer scope is unjustified and split this workflow pollutes the code (and check this condition two times is unnecessary on this same level) 
these two part must be merged